core/src/test/java/org/apache/calcite/test/RelMetadataTest.java

```diff
@@ -644,6 +644,15 @@ final RelMetadataFixture sql(String sql) {
     fixture.assertThatRowCount(is(9D), is(0D), is(10d));
   }
 
+  @Test void testRowCountSortLimitOffsetDynamic() {
+    sql("select * from emp order by ename limit ? offset ?")
+        .assertThatRowCount(is(14D), is(0D), is(Double.POSITIVE_INFINITY));
```

Comment:
minor: in these tests, instead of `is(14D)`, I think it would be a bit better to use `is(EMP_SIZE)` (as some other tests in the same class do).
Similarly, instead of `is(13D)` we could use `is(EMP_SIZE - 1)`.